lms/djangoapps/support/static/support/jsx/success.jsx

```diff
@@ -0,0 +1,48 @@
+/* global gettext */
+/* eslint one-var: ["error", "always"] */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Success({ homepageUrl, dashboardUrl, isLoggedIn }) {
+  let btnText,
+    btnUrl;
+  if (isLoggedIn) {
+    btnText = gettext('Go to my Dashboard');
+    btnUrl = dashboardUrl;
+  } else {
+    btnText = gettext('Go to edX Home');
```

Comment:
I just realized that edX should be referenced by the `PLATFORM_NAME`, e.g. in python: https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/openedx/core/djangoapps/site_configuration/context_processors.py#L15